domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientTest.java

```diff
@@ -130,6 +126,35 @@ public void configFileTenant() throws Exception {
     assertThat(config.getInteger("connectionReleaseDelay"), is(30000));
   }
 
+  @Test
+  public void testPgConnectOptionsEmpty() {
+    JsonObject conf = new JsonObject();
+    PgConnectOptions options = PostgresClient.createPgConnectOptions(conf);
+    Assert.assertEquals("localhost", options.getHost());
+    Assert.assertEquals(5432, options.getPort());
+    Assert.assertEquals("user", options.getUser());
+    Assert.assertEquals("pass", options.getPassword());
+    Assert.assertEquals("db", options.getDatabase());
+  }
+
+  @Test
+  public void testPgConnectOptionsFull() {
+    JsonObject conf = new JsonObject()
+        .put("host", "myhost")
+        .put("port", 5433)
+        .put("username", "myuser")
+        .put("password", "mypassword")
+        .put("database", "mydatabase");
+
+    PgConnectOptions options = PostgresClient.createPgConnectOptions(conf);
+    Assert.assertEquals("myhost", options.getHost());
+    Assert.assertEquals(5433, options.getPort());
+    Assert.assertEquals("myuser", options.getUser());
+    Assert.assertEquals("mypassword", options.getPassword());
+    Assert.assertEquals("mydatabase", options.getDatabase());
+  }
+
+  /* DISABLED
```

Comment:
Use `@Ignore("some reason")` https://github.com/junit-team/junit4/wiki/Ignoring-tests
to ignore a test and to give an explanation when this test can be re-enabled.
This also allows IDEs to find references/usages.
JUnit will report the number of skipped tests.

Reply:
OK (or I'll just fix it).

Reply:
It does not even compile, uses classes from old driver, eg PostgresSQLConnectionIimpl .